### PR TITLE
Dala 39 remove skyminder

### PR DESCRIPTION
**Pull request by ppuetsch**
# Pull Request Remove Skyminder
Remove all usages of the skyminderclient
## Things to do during Peer Review
Please check all boxes before the Pull Request is merged. In case you skip a box, describe in the PRs description (that means: here) why the check is skipped.
- [x] The Github Actions (including Sonarqube Gateway and Lint Checks) are green. This is enforced by Github. 
- [x] A peer-review has been executed
  - [x] The code has been manually inspected by someone who did not implement the feature
- [x] The PR actually implements what is described in the JIRA-Issue
- [ ] ~~At least one E2E Test exists testing the new feature~~
- [x] Documentation is updated as required
- [x] The automated deployment is updated if required
- [x] The new version is deployed to the dev server using this branch
  - [x] It's verified that this version actually is the one deployed (check actuator/info for branch name and commit id!)
  - [x] The new feature is manually used/tested/observed on dev server
  - [x] All implemented Social Logins have been tested manually in the UI
- [ ] ~~If any work on the UI is to be merged, those changes were also documented in the Figma~~
- [x] The local Dev stack still works: execute `startDevelopmentStack.sh`, execute npm run cypress and run Cypress Tests locally

After Merge:
- [ ] Remove all Skyminder-related Github Secrets
- [ ] Paste in the chat that local env variables may be deleted

### CODE REVIEW

File: dataland-frontend/tests/e2e/specs/company-metadata/SearchCompaniesForFrameworkDataDropdownFilter.ts [hunk 1]
```diff
@@ -18,7 +18,9 @@ before(function () {
 describe("As a user, I expect the search functionality on the /companies page to adjust to the selected dropdown filters", () => {
   it("LKSG and SFDR should be displayed in the framework dropdown even though they are not yet implemented", () => {
     cy.ensureLoggedIn();
+    cy.intercept("**/api/companies/meta-information").as("companies-meta-information");
```
Comment (SiegfriedSobkowiak) [inline dataland-frontend/tests/e2e/specs/company-metadata/SearchCompaniesForFrameworkDataDropdownFilter.ts:21]: Was hat das mit der Entfernung von Skyminder zu tun?

File: dataland-frontend/tests/e2e/specs/index.ts [hunk 2]
```diff
@@ -30,7 +30,10 @@ if (runPrepopulation) {
 }
 
 if (testGroupingDisabled || cypressTestGroup === 1 || cypressTestGroup === 102) {
-  require("./company-metadata");
+  require("./landing-page");
```
Comment (SiegfriedSobkowiak) [inline dataland-frontend/tests/e2e/specs/index.ts:33]: Ist die Umsortierung hier um die Laufzeit der einzelnen Runner auszugleichen?

Comment (ppuetsch) [reply dataland-frontend/tests/e2e/specs/index.ts:33]: Genau

File: dataland-frontend/tests/e2e/specs/company-metadata/SearchCompaniesForFrameworkDataDropdownFilter.ts (hunk 1 above)
Comment (ppuetsch) [reply dataland-frontend/tests/e2e/specs/company-metadata/SearchCompaniesForFrameworkDataDropdownFilter.ts:21]: Bei einem CI-Test gab es Timeout Errors in genau diesem Test. Meine Vermutung war, dass es eine Race-Condition ist - und dass sich die so verhindern lässt.

Der Bezug zu Skyminder ist also: Leider ist bei einem Skyminder-Testlauf dieser Test rot gewesen.

Comment (SiegfriedSobkowiak) [reply dataland-frontend/tests/e2e/specs/company-metadata/SearchCompaniesForFrameworkDataDropdownFilter.ts:21]: alles klar

**Pull request merged**